lesson.rb

```diff
@@ -1,4 +1,17 @@
 class Lesson < ActiveRecord::Base
+
+  has_many :readings, dependent: :destroy
+  belongs_to :course
+  belongs_to :in_class_assignments,
```

Comment:
Also, since it's a belongs_to, it makes more grammatical sense for the relationship to be singular.